### DISCUSSION TITLE
Bug in webcam snapshots (rotate/flip)

### DISCUSSION THREAD

**Discussion by berrystephenw** [berrystephenw/OctoText#25] [General]
If your webcam has the Flip or Rotate option selected and you want snapshots the code is currently broken. There is an exception thrown on the call to rotate/flip in ffmpeg. I'm working on it, but it might take a little time. 

The workarounds are to a) not use snapshots or b) disable any rotation/flips

**Comment by berrystephenw** [comment]
I've identified the issue in the code and will be testing the solution this evening (EST). If anyone really wants to try it sooner, send me an email and I'll send you a link you can use to manually install the fix. Eventually I'll have a release candidate but right now is not the time to set that up.
sberry@gravity-performance.com

-- this should be fixed as of release 0.1.6, please let me know if you are still having problems

**Reply by natefrog69** [reply]
Thanks for your hard work in this ;)

**Reply by berrystephenw** [reply]
You are welcome. I'm not sure that I've fixed it for everyone. Please let me know if it works for you!
